website/src/edit-word-panel.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react"
+import { HiPencilAlt } from "react-icons/hi"
+import { IoCheckmarkSharp } from "react-icons/io5"
+import {
+  unstable_Form as Form,
+  unstable_FormInput as FormInput,
+  unstable_FormLabel as FormLabel,
+} from "reakit/Form"
+import { unstable_FormSubmitButton as FormSubmitButton } from "reakit/Form"
+import { IconButton } from "./components"
+import { IconTextButton } from "./components/button"
+import * as css from "./edit-word-panel.css"
+import { useForm } from "./form-context"
+import { FormFieldsFragment } from "./graphql/dailp"
+
+/** Button that allows user to enter edit mode in the word panel, and edit fields of a word. */
+export const EditButton = () => {
+  const { form, isEditing, setIsEditing } = useForm()
+
+  return (
+    <Form {...form} className={css.form}>
+      {isEditing ? (
+        // Displays a "Cancel" button and "Save" button in editing mode.
+        <>
+          <IconButton
+            className={css.cancelButton}
+            round={false}
+            aria-label="Cancel editing the selected word and its features"
+            onClick={() => {
+              setIsEditing(false)
```

Comment:
I think we should hide the close button while editing, to prevent accidental loss of edits. If you disagree though, you should effectively "Cancel" if I click the close button instead of cancel or save.